premailer/__main__.py

```diff
@@ -112,6 +112,11 @@ def main(args):
         action="store_true",
         help="Pretty-print the outputted HTML.",
     )
+    
```

Comment:
This line has excessive whitespace. 
Blank lines should only be a single newline character.